e2e/setup-environment.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+echo "Clone mattermost"
+cd ../../
+git clone --depth 1 https://github.com/mattermost/mattermost.git
```

Comment:
What happens if a plugin dev runs the script a second time? Does git pull the latest changes?

Reply:
It will fail in cloning the repo (remain as is and will not pull the latest) with the following log but will continue until the end.
```
fatal: destination path 'mattermost' already exists and is not an empty directory.
```

Reply:
It would be nice if the script runs without an error and pulls the latest changes on a second run.

Reply:
I agree, but some may prefer to avoid introducing unintended changes to the work they're currently doing in the monorepo. This script is intended solely for the initial setup to help users get started quickly. For subsequent steps, they can choose how they want to update their setup once they're more familiar with the dependencies.